task_01.py

```diff
@@ -0,0 +1,30 @@
+#!user/bin/env python
+# -*- coding: utf-8 -*-
+"""Dividing families by table at party"""
+
+
+def get_party_stats(families, table_size=6):
+    """Return total number of guests and tables needed.
+    Arguments:
+        families: each familiy as list
+        table_size(int): size of table - 6
+    Returns:
+        total number of guests and tables needed
+    Examples:
+    GUEST_LIST = [['Jan'], ['Jill', 'Jack', 'Bob'], ['Jen', 'Alan'], ['Nick']]
```

Comment:
Reverse how you did the examples. First off they should be indented. Secondly, the 'code' has the triple cli symbol `>>>` while the result is blank, eg:

```
    Example:
        >>> 1 + 1
        2
```